MODULE.bazel

```diff
@@ -0,0 +1,2 @@
+module(name = "crowcpp", version = "1.0")
```

Comment:
current version is 1.2.1.2 (version is set on branch only)

Reply:
changed it. Thank you for noticing!